package.json

```diff
@@ -124,32 +130,32 @@
 		"media-library:storybook:start": "check-npm-client && start-storybook -c packages/media-library/.storybook -h calypso.localhost -p 3001"
 	},
 	"dependencies": {
-		"@automattic/babel-plugin-i18n-calypso": "file:./packages/babel-plugin-i18n-calypso",
-		"@automattic/babel-plugin-transform-wpcalypso-async": "file:./packages/babel-plugin-transform-wpcalypso-async",
-		"@automattic/calypso-analytics": "file:packages/calypso-analytics",
-		"@automattic/calypso-build": "file:./packages/calypso-build",
-		"@automattic/calypso-color-schemes": "file:./packages/calypso-color-schemes",
-		"@automattic/calypso-polyfills": "file:packages/calypso-polyfills",
+		"@automattic/babel-plugin-i18n-calypso": "^1.1.0",
```

Comment:
If a dependency's version is updated and it isn't changed here would this (or any other package) start getting the version it's requesting from npm instead of the local workspace? Does yarn just check if the package's version in the workspace matches and then pull it down if it's a different version? (Hopefully this isn't a totally ignorant question... I tried to look a little bit and I think what I read meant it would start pulling from NPM but I wasn't sure!)

Reply:
Correct. If `"@automattic/babel-plugin-i18n-calypso"` goes to 2.0.0, and nobody updates the dependency here, then this `@automattic/babel-plugin-i18n-calypso@^1.1.0` will be downloaded from the npm registry.

We (with @blowery ) talked about introducing some mechanism to ensure that doesn't happen, but we are not there yet. Maybe `lerna` has something to enforce it?

Reply:
That was my next question... they should definitely be locked together from the top level package.json to each individual package's version? Is that true of neighboring packages as well?